sql/lock.cc

```diff
@@ -1106,7 +1111,7 @@ bool Global_read_lock::make_global_read_lock_block_commit(THD *thd)
     If we didn't succeed lock_global_read_lock(), or if we already suceeded
```

Comment:
The original comment/assumption is no longer accurate. Same on Line 1095. (github won't allow me to comment on a line outside the change context...)